core/src/test/java/org/elasticsearch/action/update/UpdateRequestTests.java

```diff
@@ -73,8 +73,10 @@ public void testUpdateRequest() throws Exception {
 
         // script with params
         request = new UpdateRequest("test", "type", "1");
-        request.source(XContentFactory.jsonBuilder().startObject().startObject("script").field("inline", "script1").startObject("params")
-                .field("param1", "value1").endObject().endObject().endObject());
+        request.fromXContent(XContentFactory.jsonBuilder().startObject()
+            .startObject("script").field("inline", "script1")
```

Comment:
Could you write this like:
```java
request.fromXContent(XContentFactory.jsonBuilder().startObject()
  .startObject("script")
    .field("inline", "script1")
    .startObject("params")
      .field("param1", "value1")
    .endObject()
  .endObject().endObject());
```
so the structure is more obvious when you scan it?